packages/macros/src/ember-addon-main.ts

```diff
@@ -69,7 +69,7 @@ export = {
     if (!babelPlugins.some(isEmbroiderMacrosPlugin)) {
       let appInstance = this._findHost();
       let source = appOrAddonInstance.root || appOrAddonInstance.project.root;
-      babelPlugins.unshift(MacrosConfig.for(appInstance).babelPluginConfig(source));
+      babelOptions.plugins = babelPlugins.concat(MacrosConfig.for(appInstance).babelPluginConfig(source));
```

Comment:
```suggestion
      babelOptions.plugins = [
        ...MacrosConfig.for(appInstance).babelPluginConfig(source),
        ...babelOptions.plugins
      ];
```

Reply:
Or an even smaller change:

```suggestion
      babelOptions.plugins.unshift(...MacrosConfig.for(appInstance).babelPluginConfig(source));
```